tests/test_hub.py

```diff
@@ -183,3 +185,13 @@ def test_get_allowed_tasks(tasks_dict, expected):
     tasks = get_allowed_tasks(tasks_dict)
 
     assert tasks == expected
+
+
+class ValidateYaml(TestCase):
+    readme_filepaths = []
+    for glob_path in ["measurements/*/README.md", "metrics/*/README.md", "comparisons/*/README.md"]:
+        readme_filepaths.extend(glob.glob(glob_path))
+    for readme_file in readme_filepaths:
+        with open(readme_file) as f_yaml:
+            x = yaml.safe_load_all(f_yaml)
```

Comment:
Is it not necessary to split off the YAML part of the README?

Reply:
The README is loaded in delimited by `---`, so calling next() on the generator automatically gets the YAML (the first section)